lib/pages/login_page.dart

```diff
@@ -170,10 +214,10 @@ class _LoginPageState extends State<LoginPage> {
           //saving the values to our shared preferences
           await HelperFunction.saveUserLoggedInStatus(true);
           await HelperFunction.saveUserEmailSF(email);
-          await HelperFunction.saveUserNameSF(snapshot.docs[0]['fullName']);
+          await HelperFunction.saveUserNameSF(snapshot.docs[0]['FullName']);
```

Comment:
don't change the database field name. it will not work.

Reply:
fixed boss